query/src/test/resources/configuration-parsing-test-enbledInDefault.xml

```diff
@@ -15,9 +15,6 @@
       </local-cache-configuration>
       <local-cache name="default" configuration="base"/>
       <local-cache name="simple" configuration="base"/>
-      <local-cache name="not-searchable" configuration="base">
```

Comment:
This removal hides a problem. Indexed entities should not be inherited if indexed=false